server/routes/carts.js

```diff
@@ -26,6 +28,12 @@ async function createCart(req, res) {
 
 async function getCarts(req, res) {
   try {
+    const { isAdmin } = await User.findOne({ _id: req.user })
```

Comment:
@AStashevskaya I think this pattern with checking route for admin is very common, could  you create middleware `adminSessionMiddleware` which checks `jwt` token and adds `user` (all fields, not only `id`) to `req`